pkg/sql/set_cluster_setting.go

```diff
@@ -193,6 +195,18 @@ func (p *planner) getAndValidateTypedClusterSetting(
 }
 
 func (n *setClusterSettingNode) startExec(params runParams) error {
+
+	if strings.HasPrefix(n.name, "sql.defaults") {
+		params.p.BufferClientNotice(
+			params.ctx,
+			errors.WithHintf(
+				pgnotice.Newf("setting global default %s`", n.name),
```

Comment:
maybe" setting global default %s is not recommended" (remove the backtick at the end)